samples/java-protobuf-valueentity-counter/src/test/java/com/example/actions/ExternalCounterActionTest.java

```diff
@@ -32,12 +33,12 @@ public class ExternalCounterActionTest {
 
   private AutoCloseable closeable;
 
-  @Before
+  @BeforeEach
   public void setUp() {
     closeable = MockitoAnnotations.openMocks(this);
   }
 
-  @Before
```

Comment:
For the record, that was my WTF moment of the day. 

I don't know how this was working before, but the `tearDown` method should be annotated with `After`, not `Before`. 

What I'm guessing is that Junit4 will only run the first method it finds annotated with `Before`. And the `tearDown` where never called.